src/containers/RequestCodesPage/index.jsx

```diff
@@ -1,18 +1,26 @@
 import { connect } from 'react-redux';
 
 import RequestCodesPage from '../../components/RequestCodesPage';
-
```

Comment:
nit: We should keep this blank line to separate the component imports from action imports. Makes it a little more clear what things we're importing where.